COMPILING.md

```diff
@@ -87,9 +87,17 @@ git clone --recursive https://github.com/cuberite/cuberite.git
 
 ### Building ###
 
-Follow the instructions at [CMake on Unix-based platforms](#cmake-on-unix-based-platforms).
+Run the following commands to build Cuberite:
+
+```
+mkdir Release
+cd Release
+cmake -DCMAKE_BUILD_TYPE=RELEASE ..
+make -j`nproc`
```

Comment:
It runs as many jobs as possible at once. With current hardware, the bottleneck is seldom the CPU, so the number of cores is no longer that important anyway.